client/web/src/components/panels/Document.vue

```diff
@@ -6,101 +6,229 @@
+						"
+						><BlurSharpenTool
+					/></ShelfItem>
+					<ShelfItem
+						title="Relight Tool"
+						:active="activeTool === 'Relight'"
+						@click="'tool not implemented' || selectTool('Relight')"
+						><RelightTool
+					/></ShelfItem>
 
 					<ItemDivider horizontal />
 
-					<ShelfItem title="Path Tool" :active="activeTool === 'Path'" @click="'tool not implemented' || selectTool('Path')"><PathTool /></ShelfItem>
-					<ShelfItem title="Pen Tool" :active="activeTool === 'Pen'" @click="'tool not implemented' || selectTool('Pen')"><PenTool /></ShelfItem>
-					<ShelfItem title="Freehand Tool" :active="activeTool === 'Freehand'" @click="'tool not implemented' || selectTool('Freehand')"><FreehandTool /></ShelfItem>
-					<ShelfItem title="Spline Tool" :active="activeTool === 'Spline'" @click="'tool not implemented' || selectTool('Spline')"><SplineTool /></ShelfItem>
-					<ShelfItem title="Line Tool" :active="activeTool === 'Line'" @click="'tool not implemented' || selectTool('Line')"><LineTool /></ShelfItem>
-					<ShelfItem title="Rectangle Tool (M)" :active="activeTool === 'Rectangle'" @click="selectTool('Rectangle')"><RectangleTool /></ShelfItem>
-					<ShelfItem title="Ellipse Tool (E)" :active="activeTool === 'Ellipse'" @click="selectTool('Ellipse')"><EllipseTool /></ShelfItem>
-					<ShelfItem title="Shape Tool" :active="activeTool === 'Shape'" @click="'tool not implemented' || selectTool('Shape')"><ShapeTool /></ShelfItem>
+					<ShelfItem
+						title="Path Tool"
+						:active="activeTool === 'Path'"
+						@click="'tool not implemented' || selectTool('Path')"
+						><PathTool
+					/></ShelfItem>
+					<ShelfItem
+						title="Pen Tool"
+						:active="activeTool === 'Pen'"
+						@click="'tool not implemented' || selectTool('Pen')"
+						><PenTool
+					/></ShelfItem>
+					<ShelfItem
+						title="Freehand Tool"
+						:active="activeTool === 'Freehand'"
+						@click="
+							'tool not implemented' || selectTool('Freehand')
+						"
+						><FreehandTool
+					/></ShelfItem>
+					<ShelfItem
+						title="Spline Tool"
+						:active="activeTool === 'Spline'"
+						@click="'tool not implemented' || selectTool('Spline')"
+						><SplineTool
+					/></ShelfItem>
+					<ShelfItem
+						title="Line Tool"
+						:active="activeTool === 'Line'"
+						@click="'tool not implemented' || selectTool('Line')"
+						><LineTool
+					/></ShelfItem>
+					<ShelfItem
+						title="Rectangle Tool (M)"
+						:active="activeTool === 'Rectangle'"
+						@click="selectTool('Rectangle')"
+						><RectangleTool
+					/></ShelfItem>
+					<ShelfItem
+						title="Ellipse Tool (E)"
+						:active="activeTool === 'Ellipse'"
+						@click="selectTool('Ellipse')"
+						><EllipseTool
+					/></ShelfItem>
+					<ShelfItem
+						title="Shape Tool"
+						:active="activeTool === 'Shape'"
+						@click="'tool not implemented' || selectTool('Shape')"
+						><ShapeTool
+					/></ShelfItem>
```

Comment:
Please take care to not commit accidental whitespace formatting changes since it makes the code review process larger.

Reply:
Same thing with Vetur's formatter =/.